Sources/TuistKit/Commands/InitCommand.swift

```diff
@@ -194,27 +196,35 @@ class InitCommand: NSObject, Command {
     /// - Returns: Array of parsed attributes
     private func validateAttributes(_ attributes: [String: OptionArgument<String>],
                                     template: Template,
+                                    name: String,
+                                    platform: Platform,
                                     arguments: ArgumentParser.Result) throws -> [String: String] {
-        try template.attributes.reduce([:]) {
-            var mutableDict = $0
+        try template.attributes.reduce(into: [:]) {
+            if $1.name == "name" {
```

Comment:
What about naming $0 and $1? Considering this is not a one-line closure, it makes it easier to read the code.